docgen/src/examples/e-commerce/search.js

```diff
@@ -1,45 +1,69 @@
-'use strict';
 /* global instantsearch algoliasearch */
+'use strict';
 
 var search = instantsearch({
   indexName: 'instant_search',
   searchClient: algoliasearch('latency', '6be0576ff61c053d5f9a3225e2a90f76'),
-  routing: true
+  routing: true,
 });
 
+const customSearchbox = instantsearch.connectors.connectSearchBox(
+  ({ refine, widgetParams }, isFirstRender) => {
```

Comment:
Not sure about the usage of ES2015+ for those demos. IIRC they are not transpiled.

Reply:
You're right!